javaparser-core/src/main/java/com/github/javaparser/ast/visitor/EqualsVisitor.java

```diff
@@ -771,14 +771,7 @@ public Boolean visit(final ObjectCreationExpr n, final Visitable arg) {
 
     @Override
     public Boolean visit(final Name n, final Visitable arg) {
-        final Name n2 = (Name) arg;
-        if (!objEquals(n.getIdentifier(), n2.getIdentifier()))
-            return false;
-        if (!nodeEquals(n.getQualifier(), n2.getQualifier()))
-            return false;
-        if (!nodeEquals(n.getComment(), n2.getComment()))
-            return false;
-        return true;
+        return n.equals(arg);
```

Comment:
This too is generated code. You could however subclass it and override this method.

Reply:
I could subclass and overriding this method (and the others) however I don't know how I can make this code being used by the JavaParser. 

From what I see there is a private SINGLETON in EqualsVisitor, that is used in the static "equals" method of EqualsVisitor and that equals method is used in Node.equals. As the AST is build from code subclassing Node I would need to make this Node.equals method use "my" EqualsVisitor subclass, so the Node subclasses inherit the new behaviour. How do I do this?